sdk/metricsadvisor/Azure.AI.MetricsAdvisor/api/Azure.AI.MetricsAdvisor.netstandard2.0.cs

```diff
@@ -686,7 +686,7 @@ public partial class DataFeedRollupSettings
         public DataFeedRollupSettings() { }
         public string AlreadyRollupIdentificationValue { get { throw null; } set { } }
         public System.Collections.Generic.IList<string> AutoRollupGroupByColumnNames { get { throw null; } }
-        public Azure.AI.MetricsAdvisor.Models.DataFeedAutoRollupMethod? RollupMethod { get { throw null; } set { } }
+        public Azure.AI.MetricsAdvisor.Models.DataFeedAutoRollupMethod? DataFeedAutoRollupMethod { get { throw null; } set { } }
```

Comment:
curious why the decision to maintain `DataFeed` here. will `AutoRollupMethod` work too?

Reply:
`RollupMethod` doesn't make clear that this is only to be used when the rollup type is set to `AutoRollup`. It didn't occur to me, but naming it only `AutoRollupMethod` sounds better. I'll update it and check with others.